src/pages/Admin/Room/Edit/RoomEdit.tsx

```diff
@@ -32,13 +33,16 @@ const EditRoom: React.FC<EditRoomProps> = ({
     watch,
     handleSubmit,
     setValue,
+    errors,
     formState: { isSubmitting, dirty },
   } = useForm({
+    validationSchema: roomEditSchema,
```

Comment:
As mentioned elsewhere RE: using the same validation as `roomCreateSchema`

Reply:
https://github.com/sparkletown/sparkle/pull/1176#discussion_r576619116